### PR TITLE
fix: latlong image typo and prime meridian alignment

### PR DESCRIPTION
**Pull request by nakajimayoshi**
fix: latlong image typo and prime meridian alignment

## Description
Fixes the typo in the image used in the latlongfix command and re-aligned the prime meridian to cross Greenwich, U.K.

## Test Results
Before: 
image

After:
image

## Discord Username
HowNowBrownCrow#2591

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## Changelog
 
+Update <small>_ November 2022</small>
```
Comment (pdellaert) [inline .github/CHANGELOG.md:3]:
```suggestion
Update <small>_ December 2022</small>

```

File: .github/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+- fix: fix latlong image typo and prime meridian alignment (04/12/2022)
+
```
Comment (pdellaert) [inline .github/CHANGELOG.md:5]:
```suggestion
```

**Review by pdellaert — COMMENTED** [review]
Small fix on changelog. Otherwise all good!

File: .github/CHANGELOG.md (hunk 1 above)
Comment (nakajimayoshi) [reply .github/CHANGELOG.md:3]: Will fix, thanks!

**Review by pdellaert — APPROVED** [review]

**Review by oim1 — APPROVED** [review]

**Comment by tracernz** [comment]
Thanks for doing this! 👍

**Pull request merged**